_about_livecoms/index.md

```diff
@@ -15,7 +15,9 @@ practices in molecular modeling and simulation. These works are living
 documents, regularly updated with community input, and can include
 perpetual updated reviews, tutorials, comparisons between software
 packages, and other documents which aim to improve the studies in the
-field and require ongoing updates. 
+field and require ongoing updates.
+
+**Sustainability**: We believe our model for LiveCoMS will allow this journal to be around for the long haul and have significant impact on the field, and have budgeted and planned accordingly. However, as this is a publishing experiment, we are taking several safeguards to ensure research published with LiveCoMS is available for posterity. First, authors retain ownership over their work and control of their GitHub repository, allowing them to reuse it however they see fit, and archive it in any way they like. Second, the University of California library system has agreed to archive each year's publications from LiveCoMS in their eScholarship online repository, ensuring the material is archived for the long term in a way which is independent of our publishing model. 
```

Comment:
To be a bit more formal ...
"allow this journal to be around for the long haul" -> "ensure the journal's longevity"